### PR TITLE
Ci docker test

### PR DESCRIPTION
**Pull request by aradnaev**

### CODE REVIEW

File: etabotsite/default_settings.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
+{
+    "LOCAL_MODE": true,
```
Comment (cjlewis11) [inline etabotsite/default_settings.json:2]: Please add default log_filename_with_path to a local file within the home directory.

File: etabotsite/default_settings.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "DOCKER_MODE": true,
+    "local_host_url":"http://127.0.0.1:8000",
+    "prod_host_url":"http://localhost:8000",
+    "AWS_SQS_REGION": "us-west-2",
+    "CELERY_DEFAULT_QUEUE": "etabotqueue",
+    "_MESSAGE_BROKER_comment":"rabbitmq or aws",
+    "MESSAGE_BROKER":"aws",
+    "RMQ_HOST" : "localhost",
+    "RMQ_VHOST" : "etabotvhost",
+    "db": {
```
Comment (cjlewis11) [inline etabotsite/default_settings.json:12]: Please bring the default settings inline with the new naming convention for database. DB_NAME, DB_HOST, etc...

File: README.md [hunk 3]
```diff
@@ -175,8 +183,8 @@ $ pytest <path to dir with tests>
     "db": {
         "ENGINE": "django.db.backends.postgresql",
         "NAME": "your_db_name",
```
Comment (cjlewis11) [inline README.md:185]: Current convention requires DB_NAME as well and DB_HOST

File: etabotsite/default_settings.json (hunk 1 above)
Comment (aradnaev) [reply etabotsite/default_settings.json:3]: done, please review and resolve if ok

File: etabotsite/default_settings.json (hunk 2 above)
Comment (aradnaev) [reply etabotsite/default_settings.json:13]: done, please review and resolve if ok

File: README.md (hunk 3 above)
Comment (aradnaev) [reply README.md]: done, please review and resolve if ok

**Pull request merged**